testem.js

```diff
@@ -1,8 +1,7 @@
-/* eslint-env node */
+/* eslint-disable no-process-env */
```

Comment:
This is not needed. Migrated scripts already updated with this change.

Reply:
done